config/rbac/role.yaml

```diff
@@ -99,7 +99,24 @@ rules:
   - get
   - list
   - patch
-
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - '*'
+- apiGroups:
+  - argoproj.io
+  resources:
+  - workflows
```

Comment:
This is required b/c you made the watcher change but this is not desired. We have previous fix to remove clusterrole access to watch Workflows due to memory issues. See #73

Reply:
This is resolved after watching only specific namespace